Client/TeamTalkClassic/TeamTalk.rc

```diff
@@ -339,7 +339,7 @@ CAPTION "TeamTalk"
 MENU IDR_MENU_MAIN
 FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
-    LTEXT           "Channels list",IDC_STATIC_CHANLIST,0,0,0,0
+//    LTEXT           "Channels list",IDC_STATIC_CHANLIST,0,0,0,0

```

Comment:
Can you just delete this line? Otherwise I think Visual Studio will autogenerate a removal.